Makefile

```diff
@@ -1,5 +1,5 @@
 .DEFAULT_GOAL = help
-.PHONY: help install gitmodules build start stop wait-healthy sh exec logs watch lint fix test run dev prod
+.PHONY: help install gitmodules build start stop wait-healthy sh exec logs watch lint lint\:fix test run dev prod
```

Comment:
Unclear if this is a false positive:

<img width="709" alt="Screenshot 2020-03-30 at 15 58 50" src="https://user-images.githubusercontent.com/1784740/77927646-5fbfe700-729f-11ea-8457-79c781e3ce1b.png">

Reply:
@NuclearRedeye `:` allowed?

Reply:
Just had a play with creating a file called `lint:fix`, and the `.PHONY` doesn't work. Not escaping it here leads to `Makefile:2: *** multiple target patterns.  Stop.`

Reply:
Though `lint*` seems to work.

Reply:
False positive. From a quick google that rule names in Makefiles follow the same naming rules as files, but I too can't get it to accept the ':' in the PHONY target hence best I could come up with was the below...

.PHONY: lint*fix
lint\:fix:
        @touch 'lint:fix'
        @echo OK

Reply:
Thanks. I'll go with `lint*` as it covers both `lint` and `lint:fix`.

Reply:
FWIW, the current version may work on Linux and Make 4.1:
```
$ make lint:fix                                                                  
docker-compose --file .docker/docker-compose.yml --file .docker/docker-compose.dev.yml run --rm app npm run lint:fix

> @ lint:fix /app
> eslint . --fix --ext .js,.ts

[15:47:18][giorgio@Newton:~/code/article-page]$ make lint:fix
docker-compose --file .docker/docker-compose.yml --file .docker/docker-compose.dev.yml run --rm app npm run lint:fix

> @ lint:fix /app
> eslint . --fix --ext .js,.ts
```

Reply:
You did a `touch lint:fix` before that?

I'm on 3.81 here.

Reply:
Yes, messed up the copy paste from the terminal. I've tried everything with the `*` version now:
```
$ git status | grep lint
On branch linting
Your branch is up-to-date with 'thewilkybarkid/linting'.
        lint
        lint:fix
        "lint\\:fix"
```
and all the targets run correctly :+1:

Reply:
Will leave as `lint*` for compatibility.